docs/Main.md

```diff
@@ -37,6 +37,8 @@ style="max-width:100%;"></a>
 </p>
 \endhtmlonly
 
+![banner](docs/Images/banner.png)
```

Comment:
This actually prints the word "banner" under the image (and is probably the same in the README)

Reply:
Fixed, thanks for catching